locale/en.po

```diff
@@ -3437,17 +3433,16 @@ msgstr ""
 
 msgid "MICROBE_STAGE_COLLECT_TEXT"
 msgstr ""
-"Collect [thrive:compound type=\"glucose\"][/thrive:compound] (white clouds) by moving over it.\n"
-"\n"
-"Your cell needs [thrive:compound type=\"glucose\"][/thrive:compound] in order to produce energy ([thrive:compound type=\"atp\"][/thrive:compound]) to stay alive.\n"
+"To survive, a microbe will need [thrive:compound type=\"atp\"][/thrive:compound]. For your species, that means finding [thrive:compound type=\"glucose\"][/thrive:compound].\n"
```

Comment:
I think this is a regression in understandability as this uses the more complex technical term before mentioning energy.